### DISCUSSION TITLE
N Value because of refelction

### DISCUSSION THREAD

**Discussion by nyunity** [jomjol/AI-on-the-edge-device#2800] [Q&A]
Hi all,

unfortunaly, i have a reflection in the middle of the numbers, see attached file.

Someone has a workaround or some good ideas to prevent it?

My Watermeter works well without any problems.

Thanks!
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/8504397/74165d18-4b7a-4640-b6a6-ddab68b5c849)

**Comment by signrossi** [comment]
your setup is camera direct or with mirror?

my is with mirror and i fiddled it so that the refelection-spot goes under the numbers- also at yours- there is enough space above the numbers.

basically i think it is an issue with the onboard LED, i also tried various settings of LED intensity- around 20-30
